hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/utils/BackgroundService.java

```diff
@@ -88,6 +88,7 @@ public void runPeriodicalTaskNow() throws Exception {
 
   // start service
   public void start() {
+    LOG.info("Starting up service {}", serviceName);
```

Comment:
is this needed?

Reply:
This is leftover from testing. Idea is to remove it before PR creation, but I can do that now as well.